test/multifile/constant-struct-with-padding/Other.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: true
```

Comment:
Ah, cool cool. My theory is that it isn't needed, but I might be wrong. I sent up https://github.com/apple/swift/pull/4864 to remove the other instances of `RUN: true`, let's see what more knowledgeable people have to say.